src/output-json.c

```diff
@@ -150,6 +150,9 @@ static enum JsonOutput json_out = ALERT_FILE;
 
 static enum JsonFormat format = COMPACT;
 
+/* Prepend @cee: cookie to eve JSON syslog output for CEE/Lumberjack compatibility */
+static int syslog_cee_format = FALSE; /* Default OFF */
```

Comment:
I don't like that we're adding another global here. I know @regit had some work to remove all globals from the json output, but it had some issues so it wasn't merged yet.
The reason globals are bad here is that we can have multiple instances of the json output, so settings should be per instance.